python/llm/example/GPU/vLLM-Serving/README.md

```diff
@@ -0,0 +1,109 @@
+# vLLM continuous batching on Intel GPUs (experimental support)
+
+This example demonstrates how to serve a LLaMA2-7B model using vLLM continuous batching on Intel GPU (with BigDL-LLM low-bits optimizations).
+
+The code shown in the following example is ported from [vLLM](https://github.com/vllm-project/vllm/tree/v0.2.1.post1).
+
+## Example: Serving LLaMA2-7B using Xeon GPU
+
+In this example, we will run Llama2-7b model using Arc A770 and provide `OpenAI-compatible` interface for users.
+
+### 0. Environment
+
+To use Intel GPUs for deep-learning tasks, you should install the oneAPI Base Toolkit from [here](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html) first.
```

Comment:
need to specify which version

Reply:
Fixed. We now use this [page](https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/GPU#requirements) to guide user for driver/oneAPI Base Toolkit installation.